.github/workflows/test.yml

```
@@ -175,6 +174,11 @@ jobs:
     if: needs.config.outputs.shouldUpdateSnapshots == 'true'
     steps:
       - uses: actions/checkout@v4
+        with:
+          # NOTE: デフォルトの設定だとgithub-push-actionが動いてくれないので設定を変える。
```

変えるべきと見えなくもないので、変えたということがわかるように
```suggestion
          # NOTE: デフォルトの設定だとgithub-push-actionが動いてくれないので設定を変えている。
```